Apromore-Custom-Plugins/File-Portal-Plugin/src/main/resources/META-INF/spring/context.xml

```diff
@@ -75,4 +75,11 @@
         <beans:bean class="org.apromore.plugin.portal.file.DeleteSelectionPlugin"/>
     </osgi:service>
 
+    <osgi:service interface="org.apromore.plugin.portal.WebContentService">
+        <beans:bean class="org.apromore.plugin.portal.BundleWebContentService">
+            <beans:constructor-arg ref="bundleContext"/>
+            <beans:property name="include" value=".*\.svg"/>
+        </beans:bean>
+    </osgi:service>
+
```

Comment:
Why the resource location pattern here doesn't need any key identifying the plugin as shown in other plugins, like "loganimation/icon.svg", "merge/icon.svg", "/similaritysearch/icon.svg"

Reply:
The icons in the file portal plugin all have unique names (e.g. model-edit.svg, upload-file.svg) rather than ever being called just icon.svg.  It's a bit of an artifact of them being originally part of the Portal rather than starting their lives as a plugin.  It wouldn't be any great effort to copy them into a file/ subdirectory to use the same convention as the others to avoid contention.